6_Xenium_sample_analysis/Niches_analysis/xenium_healthy_gut_add_on_niches.ipynb

```diff
@@ -155,11 +155,11 @@
   },
```

Comment:
<p>It would be good to clean the paths. </p><p> </p><p>You could create a folder inside your repository, say <code>data</code>, add it to your <code>.gitignore</code>, and softlink all your input files there.</p><p>Then your reading path will be <code>"../data/enium_add_on_celltypes_and_stem_cells_types_without_image.h5ad"</code>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>